Makefile

```diff
@@ -167,7 +167,3 @@ propagate-version:
 	done
 
 .PHONY: source install clean check variables
-
-# implicit rule/recipe for man page creation
-%.1: %.rst
-	$(RST2MAN) $< $@
```

Comment:
Wouldn't it be better to move the `$(PYTHON) setup.py man` rule here and keep the prerequisite `man/avocado.1`? This way the man page would need to be regenerated if the respective `rst` is changed.

Reply:
> Hi @ana , this looks good, but I have a question for you before my ack. Thanks!

Thanks @clebergnu, I made an update to the PR.